doc/plugins/limit-req.md

```diff
@@ -37,7 +40,7 @@ limit request rate using the "leaky bucket" method.
 |---------     |--------|-----------|
 |rate          |required|is the specified request rate (number per second) threshold. Requests exceeding this rate (and below `burst`) will get delayed to conform to the rate.|
 |burst         |required|is the number of excessive requests per second allowed to be delayed. Requests exceeding this hard limit will get rejected immediately.|
-| key          |required|is the user specified key to limit the rate, now accept those as key: "remote_addr"(client's IP), "server_addr"(server's IP), "X-Forwarded-For/X-Real-IP" in request header.|
+| key          |required|is the user specified key to limit the rate, now accept those as key: "remote_addr"(client's IP), "server_addr"(server's IP), "X-Forwarded-For", "X-Real-IP/consumer_name(consumer's ID)" in request header.|
```

Comment:
Is consumer_name or consumer_id?
I'm confused.

Reply:
The values of "consumer_name" and "consumer_id" here are the same. This is to show that these two values are equivalent.

Reply:
fixed.

Reply:
If they are the same, why use two different names? Is it id or name?

Reply:
> @Firstsawyou conflicted, please rebase your branch

fixed

Reply:
Here is my incomplete consideration, I have changed to the name method uniformly.

Reply:
This is to determine a version first, and later will be resolved by upgrading the version. This is a related to issue: #2275